test/utils/daemon-factory.js

```diff
@@ -2,22 +2,20 @@
 
 const { createFactory } = require('ipfsd-ctl')
 const isNode = require('detect-node')
-let ipfsModule
-let ipfsHttpModule
-
-// webpack requires conditional includes to be done this way
-if (process.env.IPFS_JS_MODULE) {
-  ipfsModule = require(process.env.IPFS_JS_MODULE)
-} else {
-  ipfsModule = require('ipfs')
-}
 
-if (process.env.IPFS_JS_HTTP_MODULE) {
+let ipfsHttpModule
+let ipfsModule
+try {
   ipfsHttpModule = require(process.env.IPFS_JS_HTTP_MODULE)
-} else {
+} catch (err) {
```

Comment:
We should be able to use [optional catch binding](https://2ality.com/2017/08/optional-catch-binding.html) now?

```suggestion
} catch {
```